### PR TITLE
Update QQ to 8.9.1.

### PR DESCRIPTION
**Pull request by hillwoodroc**
Update QQ

### CODE REVIEW

File: src/winetricks [hunk 1]
```diff
@@ -6,7 +6,7 @@
 
 # Name of this version of winetricks (YYYYMMDD)
 # (This doesn't change often, use the sha256sum of the file when reporting problems)
-WINETRICKS_VERSION=20170316
+WINETRICKS_VERSION=20170323
```
Comment (austin987) [inline src/winetricks]: Please don't bump WINETRICKS_VERSION, leave that to me.

File: src/winetricks [hunk 2]
```diff
@@ -16338,20 +16338,20 @@ load_popfs()
...
  file2="QQ.tar.gz"\
  installed_exe1="$W_PROGRAMS_X86_WIN/Tencent/QQ/Bin/QQScLauncher.exe" \
  homepage="http://www.qq.com" \
  unattended="no"
 
 load_qq()
 {
-    w_download https://dldir1.qq.com/qqfile/qq/QQ8.0/16968/QQ8.0.exe 3af11db606ea9f8358db5c9eb12398da3464de020b7da7cbf3e13662ee33d228
-    w_download http://hillwoodhome.net/wine/QQ.tar.gz eb5cd6371eb75ec9e2fc0271199df05cbb9f38a60c2e81d5d8ac7daeb40aba62
+    w_download http://dldir1.qq.com/qqfile/qq/QQ8.9.1/20437/QQ8.9.1.exe f883a26a398deeb128fd3f5a0bd6f5440340938a
```
Comment (austin987) [inline src/winetricks]: You removed two important changes:
* HTTPS
* SHA256 instead of SHA1

File: src/winetricks [hunk 3]
```diff
@@ -16338,20 +16338,20 @@ load_popfs()
...
+    w_download http://hillwoodhome.net/wine/QQ.tar.gz 08de45d3e5bb34b22e7c33e1163daec69742db58
```
Comment (austin987) [inline src/winetricks]: Same here (SHA256)

**Comment by hillwoodroc** [comment]
OK.

**Pull request closed without merging**